README.md

```diff
@@ -17,6 +17,54 @@ The three models are schematically depicted in the animation below
 
 The models are trained on modern reanalysis ERA5 which resolves part of the atmospheric gravity wave spectrum. Since it does not resolve the mesoscale wave spectrum, the repository provides some functionality to retrain parts of the models M1-M3 trained on ERA5 to be retrained on fluxes obtained from a 1 km global IFS model which resolves the whole mesoscale wave spectrum.
 
+## Getting Started
+
+### Obtaining `nonlocal_gwfluxes`
+
+Clone the latest version of `nonlocal_gwfluxes` using the following command
+
+```bash
+git clone git@github.com:DataWaveProject/nonlocal_gwfluxes.git
+```
+
+### Install Dependencies
+
+This project uses `poetry` to manage dependencies. To install the dependencies we first need poetry.
+
+```bash
+cd nonlocal_gwfluxes
+python -m venv .nlgw
+```
+
+This will create a Python virtual environment inside our repository. Before we use this environment we must first `activate` it.
+
+```bash
+source .nlgw/bin/activate
```

Comment:
This won't be the case for windows machines.

Reply:
:laughing: yeah, I do tend to omit windows users :eyes:. If @amangupta2 is happy to just have linux instructions for now, then we can proceed.

Reply:
![poetry_torch_install_failed](https://github.com/user-attachments/assets/b02bce67-bcb0-4f3f-ab86-d9c57854e56a)

Seems like poetry is using a lot of space on Sherlock. So, I updated my conda, created a new conda environment, pulled the add-pyprojecttoml branch to my local repo and executed the poetry command stated above, and it shot up my usage (by atleast 3 gigabytes) until there was no space left on my home dir. Does torch really take up THAT MUCH space? I doubt. Do you have an idea what may be going on?

Reply:
Also, it has been a while since I thought about windows. I don't think we will lose any audience by restricting ourselves to linux/unix.

Reply:
ok looks like we all agree. I won't explicitly consider windows support when writing docs